runtime/src/lib.rs

```diff
@@ -132,8 +132,8 @@ pub mod opaque {
 // https://docs.substrate.io/main-docs/build/upgrade#runtime-versioning
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-    spec_name: create_runtime_str!("nh-node"),
```

Comment:
Ok, we should understand if we want to re-genesis our chain. @DanieleDiBenedetto has told me that maybe we want.... maybe is better to decide what we want to do ASAP :smile:

Reply:
Let's proceed without that for the moment, so we unlock other refactoring activities. When we will have a feedback we could open another PR.